### DISCUSSION TITLE
Merge feature/CommandLanguage branch into master

### DISCUSSION THREAD

**Discussion by Levi-Armstrong** [tesseract-robotics/tesseract#471] [General]
@mpowelson: I wanted to start the discussion about merging the `feature/CommandLanguage` branch into master. I believe we are ready to merge but want to check before starting the process.

**Comment by mpowelson** [comment]
#418 is the main thing. While I would say its likely we continue to modify the interface to the process managers, etc it has been a while since  we changed the command language itself. So I'd say we go ahead and merge it. #339 has some other things, but the only one that I would say needs to happen before the merge is that we should increment and tag the version.
